frontend/lib/resolution/resolution-types.cpp

```diff
@@ -408,6 +408,11 @@ CallInfo CallInfo::createWithReceiver(const CallInfo& ci,
                   std::move(newActuals));
 }
 
+CallInfo CallInfo::copyAndRename(const CallInfo &ci, UniqueString rename) {
+  return CallInfo(rename, ci.calledType(), ci.isMethodCall(),
+                  ci.hasQuestionArg_, ci.isParenless_, ci.actuals_);
+}
+
```

Comment:
Oh, this is nice -- so now `atomic` calls really look like `atomic` calls, and not `chpl__atomicType`?

Reply:
Yes, this is a borrowed strategy from the production compiler where we resolve a helper function defined in an internal module. For some more context, ``chpl__atomicType`` is also responsible for deciding between processor and network atomics.